### ISSUE TITLE
Buggy back-to-top button on iOS

### ISSUE THREAD

**Issue by PaulvdDool** [smashingmagazine/redesign#314]
**- Do you want to request a *feature* or report a *bug*?**
Feature (or rather an enhancement)

**- What is the current behavior?**
The back to top button hides on mobile when I'm scrolling to the top and only shows when I'm scrolling down.

**- What is the expected behavior?**
I expect the behavior to be reverted. That the button hides when I'm scrolling down and shows when I'm scrolling up. As a fairly recent article of the Nielson Norman Group also states: when I'm scrolling down I'm still reading. Only when I'm scrolling up do I have the intention of going up and might I need some assistance in reaching the top fast.
https://www.nngroup.com/articles/back-to-top/

**- Please mention your operating system version and the version of your browser**
iPhone 6
iOS 11
Chrome mobile browser

**Comment by vitalyfriedman** [comment]
Dear @PaulvdDool — that's weird, because actually the button should... never really disappear. Will be looking into it. Good catch!

**Comment by indysigner** [comment]
Yes, the back to top button is still a bit buggy on iOS Safari, but as Vitaly said, it should always be there, not just when scrolling up or down. Let me rephrase the title of this issue since it's a bit misleading.
